lib/config/schema.js

```diff
@@ -6,260 +6,709 @@
+  },
+  {
+    name: 'captureHeaders',
+    type: 'boolean',
+    defaultValue: true,
+    envVar: 'ELASTIC_APM_CAPTURE_HEADERS',
+  },
+  {
+    name: 'centralConfig',
+    type: 'boolean',
+    defaultValue: true,
+    envVar: 'ELASTIC_APM_CENTRAL_CONFIG',
+  },
+  {
+    name: 'cloudProvider',
+    type: 'select(auto,gcp,azure,aws,none)',
+    defaultValue: 'auto',
+    envVar: 'ELASTIC_APM_CLOUD_PROVIDER',
+  },
+  {
+    name: 'containerId',
+    type: 'string',
+    defaultValue: undefined,
+    envVar: 'ELASTIC_APM_CONTAINER_ID',
+  },
+  {
+    name: 'contextPropagationOnly',
+    type: 'boolean',
+    defaultValue: false,
+    envVar: 'ELASTIC_APM_CONTEXT_PROPAGATION_ONLY',
+  },
+  {
+    name: 'customMetricsHistogramBoundaries',
+    type: 'sortedNumberArray',
+    // Exponential powers-of-2 bucket boundaries, rounded to 6 significant figures.
+    //    2**N for N in [-8, -7.5, -7, ..., 16, 16.5, 17]
+    // https://github.com/elastic/apm/blob/main/specs/agents/metrics-otel.md#histogram-aggregation
+    defaultValue: [
+      0.00390625, 0.00552427, 0.0078125, 0.0110485, 0.015625, 0.0220971,
+      0.03125, 0.0441942, 0.0625, 0.0883883, 0.125, 0.176777, 0.25, 0.353553,
+      0.5, 0.707107, 1, 1.41421, 2, 2.82843, 4, 5.65685, 8, 11.3137, 16,
+      22.6274, 32, 45.2548, 64, 90.5097, 128, 181.019, 256, 362.039, 512,
+      724.077, 1024, 1448.15, 2048, 2896.31, 4096, 5792.62, 8192, 11585.2,
+      16384, 23170.5, 32768, 46341, 65536, 92681.9, 131072,
+    ],
+    envVar: 'ELASTIC_APM_CUSTOM_METRICS_HISTOGRAM_BOUNDARIES',
+  },
+  {
+    name: 'disableInstrumentations',
+    type: 'stringArray',
+    defaultValue: [],
+    envVar: 'ELASTIC_APM_DISABLE_INSTRUMENTATIONS',
+  },
+  {
+    name: 'disableMetrics',
+    type: 'stringArray',
+    defaultValue: [],
+    envVar: 'ELASTIC_APM_DISABLE_METRICS',
+  },
+  {
+    name: 'disableMetricsRegExp',
+    type: 'wildcardArray',
+    defaultValue: [],
+    deps: ['disableMetrics'],
+  },
+  {
+    name: 'disableSend',
+    type: 'boolean',
+    defaultValue: false,
+    envVar: 'ELASTIC_APM_DISABLE_SEND',
+  },
+  {
+    name: 'elasticsearchCaptureBodyUrls',
+    type: 'stringArray',
+    defaultValue: [
+      '*/_search',
+      '*/_search/template',
+      '*/_msearch',
+      '*/_msearch/template',
+      '*/_async_search',
+      '*/_count',
+      '*/_sql',
+      '*/_eql/search',
+    ],
+    envVar: 'ELASTIC_APM_ELASTICSEARCH_CAPTURE_BODY_URLS',
+  },
+  {
+    name: 'elasticsearchCaptureBodyUrlsRegExp',
+    type: 'wildcardArray',
+    defaultValue: [],
+    deps: ['elasticsearchCaptureBodyUrls'],
+  },
+  {
+    name: 'environment',
+    type: 'string',
+    defaultValue: 'development',
+    envVar: 'ELASTIC_APM_ENVIRONMENT',
+  },
+  {
+    name: 'errorOnAbortedRequests',
+    type: 'boolean',
+    defaultValue: false,
+    envVar: 'ELASTIC_APM_ERROR_ON_ABORTED_REQUESTS',
+  },
+  {
+    name: 'exitSpanMinDuration',
+    type: 'durationMiliseconds',
```

Comment:
```suggestion
    type: 'durationMilliseconds',
```

and related references.